.github/workflows/builds-review.yaml

```diff
@@ -0,0 +1,49 @@
+name: Review builds
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  build:
+    if: |
+      github.event_name == 'pull_request' &&
+      github.event.label.name == 'build::review'
+    strategy:
+      matrix:
+        include:
+          - runner: ubuntu-latest
+            subdir: linux-64
+          - runner: macos-latest
+            subdir: osx-64
+          - runner: windows-latest
+            subdir: win-64
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+          clean: true
+          fetch-depth: 0
```

Comment:
Why 0? AFAIK this means that the entire repo history (for all branches) is cloned, is this necessary?

Reply:
I'm not sure, this comes from the previous canary action, the last commit should suffice, right?

Reply:
ah yeah for both of these workflows I'm pretty sure the defaults for the `actions/checkout` action are all we need

Reply:
@jezdez I still think we can remove the entire `with` clause from both of these workflows as the defaults should be sufficient